backend/src/test/java/dev/tripdraw/presentation/controller/MemberControllerTest.java

```diff
@@ -74,4 +83,33 @@ void setUp() {
                 .then().log().all()
                 .statusCode(BAD_REQUEST.value());
     }
+
+    @Test
+    void 사용자를_조회한다() {
+        // given
+        Member member = memberRepository.save(new Member("통후추"));
+
+        // when
+        ExtractableResponse<Response> response = RestAssured.given().log().all()
+                .contentType(MediaType.APPLICATION_JSON_VALUE)
+                .when().get("/members/" + member.id())
+                .then().log().all()
+                .statusCode(OK.value())
+                .extract();
+        MemberSearchResponse memberSearchResponse = response.as(MemberSearchResponse.class);
+
+        // expect
+        assertThat(memberSearchResponse.nickname()).isEqualTo("통후추");
+    }
```

Comment:
URI 만들 때, 문자열 합쳐서 넣는 것보다, 파라미터로 넘겨주는 건 어떨까요?
```suggestion
    @Test
    void 사용자를_조회한다() {
        // given
        Member member = memberRepository.save(new Member("통후추"));

        // when
        ExtractableResponse<Response> response = RestAssured.given().log().all()
                .contentType(MediaType.APPLICATION_JSON_VALUE)
                .when().get("/members/{memberId}", member.id())
                .then().log().all()
                .statusCode(OK.value())
                .extract();
        MemberSearchResponse memberSearchResponse = response.as(MemberSearchResponse.class);

        // expect
        assertThat(memberSearchResponse.nickname()).isEqualTo("통후추");
    }
```